lib/salus/scanners/base.rb

```diff
@@ -76,11 +79,24 @@ def run!(salus_report:, required:, pass_on_raise:, reraise:)
       salus_report.add_scan_report(@report, required: required)
 
       begin
-        @report.record { run }
+        @report.record do
+          Timeout.timeout(max_lifespan / 1000.to_f) { run }
```

Comment:
best to not `run` in a Timeout block if no timeout is specified.

Reply:
Since the timeout [is defined as a default config value](https://github.com/coinbase/salus/pull/403/files/8f2aecf4fa15e4796d8328c4bc7bbf30ec93a386#diff-039372a0d92ae738bcced2074de46eb7f947085cd61a675a2e7aceb4b41cfb7e), a timeout is always specified.